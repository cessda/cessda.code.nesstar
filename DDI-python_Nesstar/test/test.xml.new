<?xml version='1.0' encoding='utf-8'?>
<codeBook xmlns="http://www.icpsr.umich.edu/DDI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2.2" ID="test" xml-lang="en" xsi:schemaLocation="http://www.icpsr.umich.edu/DDI http://www.icpsr.umich.edu/DDI/Version1-2-2.xsd">
  <docDscr>
    <citation>
      <titlStmt>
        <titl>
          New Study 2
        </titl>
      </titlStmt>
      <prodStmt>
        <software version="4.0.10" date="2018-05-02">
          Nesstar Publisher
        </software>
      </prodStmt>
    </citation>
  </docDscr>
  <stdyDscr>
    <citation>
      <titlStmt>
        <titl>
          New Study 2 
        </titl>
        <IDNo agency="GESIS">
          test
        </IDNo>
      </titlStmt>
      <rspStmt>
        <AuthEnty affiliation="tttt">
          tttt
        </AuthEnty>
      </rspStmt>
      <prodStmt>
        <software version="4.0.10" date="2018-05-02">
          Nesstar Publisher
        </software>
      </prodStmt>
    </citation>
  </stdyDscr>
  <dataDscr />
</codeBook>