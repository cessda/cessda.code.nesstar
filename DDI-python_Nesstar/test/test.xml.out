#############################################################
### modify_ddi.py 
### Python script to modify a DDI xmlfile 
### Version 1.0 2020-11-12 
### Author: W. Zenk-Möltgen 
###
### 2021-03-24 12:09:33
###
#############################################################
XML file is test.xml
Output file is test.xml.out
Errorfile is test.xml.err

XML file uses standard namespace http://www.icpsr.umich.edu/DDI
XML file uses elements with no prefix 
Start modification 
agency added to IDNo 

Result: OK 
